android/app/src/main/java/com/gutenberg/MainApplication.java

```diff
@@ -128,6 +128,9 @@ public void editorDidEmitLog(String message, LogLevel logLevel) {
             @Override
             public void performRequest(String path, Consumer<String> onSuccess, Consumer<Bundle> onError) {}
 
+            @Override
+            public void onAddMention(Consumer<String> onSuccess) {}
```

Comment:
Could we implement a "stub" version of this that always returns an user called `admin` for example?

Reply:
Done!